### ISSUE TITLE
In the README: is Miniconda still preferred for a new install?

### ISSUE THREAD

**Issue by Huite** [mamba-org/mamba#1419]
The current README states:

> ## Installation
> It's advised to install mamba from conda-forge. If you already have conda, install mamba into the base environment:
>
> `conda install mamba -n base -c conda-forge`
>
> otherwise it's best to start with Miniconda. If you want to experiment with the latest software, you can also try micromamba (more below).
>
> Another alternative (if you want/need to avoid Anaconda entirely) is to use Mambaforge via the Miniforge distribution.

Is this still up-to-date? Installing mamba with conda if you already have conda installed makes sense to me, but for a fresh install, mambaforge is quite convenient and it's what I personally recommend to my colleagues.

Worth nothing too that the miniforge page just lists the different options, but doesn't recommend a specific one:
https://github.com/conda-forge/miniforge

To sum up:
1. Is miniconda still preferred for a new install?
2. If so, by which reasoning?
3. Either way, a single paragraph of guidance (or links to relevant guidance) would probably be nice to have in the README.

**Comment by wolfv** [comment]
I would recommend either `mambaforge` (from the `miniforge` link you posted) or `micromamba`.
We should update the readme, indeed.
